src/lib/userStorage/FeedStorage.js

```diff
@@ -564,9 +564,15 @@ export class FeedStorage {
     }
 
     const isContract = await this.wallet.wallet.eth.getCode(address)
+
+    // because of a possible bug when handleReceipt creates the feed-item for receiving a bridge transaction,
+    // bridge check might not be done correctly causing a unneccessary check for contract name
+    // so we do it here again
+    const isBridgeReceive = this.wallet.getBridgeAddresses().includes(feedEvent.data.receiptEvent.from?.toLowerCase())
+
     const isBridgeOrBuy = [TxType.TX_BRIDGE_IN, TxType.TX_BRIDGE_OUT, TxType.TX_BUYGD].includes(feedEvent.txType)
 
-    if (isContract !== '0x' && !isBridgeOrBuy) {
+    if (isContract !== '0x' && !isBridgeOrBuy && !isBridgeReceive) {
```

Comment:
what is the difference between isBridgeReceive and TX_BRIDGE_IN ? isnt it supposed to be the same, seems like a double check to me

Reply:
Through various channels I explain there is a bug in the creation of bridge receive feed items, which are wrongfully created as regular receive (So not as TX_BRIDGE_IN).

This is a hotfix, but I will spend the time now on fixing the bug that causes it

Reply:
Okay. so after debugging it seems this is only related to older tx-items in the feed which are defined as regular TX_RECEIVE types.
For new feed-items it is set as expected: TX_BRIDGE_IN

Since we also decided to only run this when creating the items, this extra check does not seem to be needed anymore.
Removed